src/react-apps/applications/shared/src/components/AltinnSpinner.tsx

```diff
@@ -32,7 +33,7 @@ class AltinnSpinner extends React.Component<IAltinnSpinnerComponentProvidedProps
   public render() {
     const { classes } = this.props;
     return (
-      <div>
+      <div className={classNames(this.props.className)}>
```

Comment:
Since this is an inline styling object from this.props, I suggest style={this.props.styleObj}

Reply:
fixed🖌

Reply:
nvm has to be this way